app/src/main/java/com/gacode/relaunchx/ReLaunch.java

```diff
@@ -1566,14 +1574,16 @@ public boolean onSingleTapConfirmed(MotionEvent e) {
 							200, 200, 0);
 					gv.dispatchTouchEvent(ev);
 				} else { // other devices
+					if (!prefs.getBoolean("doNotHyph", true)) {
+						gv.setSelection(getFirstItemToScrollUp(gv));
```

Comment:
What's the point of setSelection() here if later on it is amended by the old code?

Reply:
I've forget to add "return true;" at the end of the if conditional.